Docker CI build is broken

The docker CI build is broken:
```
Successfully built a8738a81aec4
Successfully tagged staging_dut:latest
Image for service dut was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
Creating staging_dut_1 ... 
Creating staging_coordinator_1 ... 

ERROR: for staging_dut_1  "host" network_mode is incompatible with port_bindings
Creating staging_coordinator_1 ... done
Creating staging_exporter_1    ... 
Creating staging_exporter_1    ... done
[7921] Failed to execute script docker-compose

ERROR: for dut  "host" network_mode is incompatible with port_bindings
Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose/cli/main.py", line 80, in main
  File "compose/cli/main.py", line 192, in perform_command
  File "compose/metrics/decorator.py", line 18, in wrapper
  File "compose/cli/main.py", line 1165, in up
  File "compose/cli/main.py", line 1161, in up
  File "compose/project.py", line 708, in up
  File "compose/parallel.py", line 106, in parallel_execute
  File "compose/parallel.py", line 204, in producer
  File "compose/project.py", line 694, in do
  File "compose/service.py", line 563, in execute_convergence_plan
  File "compose/service.py", line 479, in _execute_convergence_create
  File "compose/parallel.py", line 106, in parallel_execute
  File "compose/parallel.py", line 204, in producer
  File "compose/service.py", line 477, in <lambda>
  File "compose/service.py", line 456, in create_and_start
  File "compose/service.py", line 333, in create_container
  File "compose/service.py", line 937, in _get_container_create_options
  File "compose/service.py", line 1069, in _get_container_host_config
  File "docker/api/container.py", line 598, in create_host_config
  File "docker/types/containers.py", line 339, in __init__
docker.errors.InvalidArgument: "host" network_mode is incompatible with port_bindings
Error: Process completed with exit code 255.
```

--- a/dockerfiles/staging/docker-compose.yml
+++ b/dockerfiles/staging/docker-compose.yml
@@ -48,5 +48,3 @@ services:
     build:
       context: "./dut"
     network_mode: "host"
-    ports:
-      - 2222:2222

